services/api-server/tests/unit/api_solvers/test_api_routers_solvers_jobs.py

```diff
@@ -313,8 +313,8 @@ async def test_run_solver_job(
 
     example = next(
         e
-        for e in ServiceDockerData.Config.schema_extra["examples"][::-1]
-        if "boot" in e["description"]
+        for e in ServiceDockerData.Config.schema_extra["examples"]
+        if "boot-options" in e
```

Comment:
interesting one. so this was not a problem before?

Reply:
no